test-main.js

```diff
@@ -0,0 +1,24 @@
+var allTestFiles = [];
+var testRegex = /(spec|test)\.js$/i;
+
+// Get a list of all the test files to include
+Object.keys(window.__karma__.files).forEach(function(file) {
+	if (testRegex.test(file)) {
+		// Normalize paths to RequireJS module names.
+		// If you require sub-dependencies of test files to be loaded as-is (requiring file extension)
+		// then do not normalize the paths
+		var normalizedTestModule = file.replace(/^\/base\/|\.js$/g, '');
```

Comment:
ah, missed these var statements not being at the top of the function. I'm assuming we follow that rule for unit tests as well. This can be fixed later.